### DISCUSSION TITLE
Making font for note list smaller

### DISCUSSION THREAD

**Discussion by glow12121** [gsantner/markor#2463] [Q&A]
Is it possible to make the file list font smaller? On my 4.3 inch phone the list entries look too big.

Also, is it possible to hide files with a dot (e. g., `.ttxfolder`)?

![Screenshot_20241028-120633_Markor](https://github.com/user-attachments/assets/bfcede17-21dc-4218-a1b8-185d788c4296)

**Comment by randomcodepanda** [comment]
>Also, is it possible to hide files with a dot (e. g., .ttxfolder)?

On the icon next to the folder on the upper right corner, you can disable the dotfiles to be visible.

**Reply by glow12121** [reply]
Great, thank you, @randomcodepanda .
